perun-core/pom.xml

```diff
@@ -198,6 +198,14 @@
 			<groupId>com.oracle.database.nls</groupId>
 			<artifactId>orai18n</artifactId>
 		</dependency>
+        <dependency>
+            <groupId>org.reflections</groupId>
+            <artifactId>reflections</artifactId>
+        </dependency>
+		<dependency>
```

Comment:
duplicated dependency, also maybe move it to "OTHERS" section of the pom file as it is not Oracle's package